.github/actions/file-diff/index.js

```diff
@@ -110,23 +110,26 @@ async function run() {
 				const md = ["", `#### ${name}`, ""];
 				const data = [name, bytesToSize(totalDiffSize)];
 
+				if (totalDiffSize - totalSize === 0) return;
```

Comment:
If there is no change in filesize from previous to now, we don't want to include it in the table (unnecessary noise)